platform-ui-skin/src/main/webapp/skin/less/component/Portal/UIEditInlineWS/Style.less

```diff
@@ -149,6 +149,10 @@ Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
 			background-position: 5px top;
 			height: 41px;
 		}
+		.SimpleTableContainerLayout {
```

Comment:
we should if we can put it under [UIContainerList/Stylesheet.css](https://github.com/Meeds-io/gatein-portal/pull/476/files#diff-f3070c237c844825eb293f4c2b33c129f732cf495c9a5c88b6dc3807b6a9a530)